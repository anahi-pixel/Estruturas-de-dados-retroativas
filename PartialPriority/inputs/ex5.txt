1 6 600
1 7 700
1 8 800
1 1 100
1 2 200
1 3 300
1 4 400
7
8
5
6
3 3
7
8