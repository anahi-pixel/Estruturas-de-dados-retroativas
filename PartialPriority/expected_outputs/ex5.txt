1 Fila de prioridade no instante atual: 
100 200 300 400 600 700 800 
         1:0 0 100
      1:0 0 2147483647
         2:0 0 200
   1:0 0 2147483647
         3:0 0 300
      3:0 0 2147483647
         4:0 0 400
1:0 0 2147483647
      6:0 0 600
   6:0 0 2147483647
         7:0 0 700
      7:0 0 2147483647
         8:0 0 800

100
7
2 Fila de prioridade no instante atual: 
100 200 400 600 700 800 
         1:0 0 100
      1:0 0 2147483647
         2:0 0 200
   1:0 0 2147483647
      4:0 0 400
1:0 0 2147483647
      6:0 0 600
   6:0 0 2147483647
         7:0 0 700
      7:0 0 2147483647
         8:0 0 800

